testsrc/main/mondrian/xmla/XmlaBasicTest.java

```diff
@@ -744,12 +744,12 @@ public void testExecuteAliasWithSharedDimension()
             + "<Schema name=\"foodmart-xmla-alias-bug\">\n"
             + "  <Dimension name=\"Customers\">\n"
             + "    <Hierarchy hasAll=\"true\" allMemberName=\"All Customers\" primaryKey=\"customer_id\">\n"
-            + "      <Table schema=\"foodmart\" name=\"customer\"/>\n"
+            + "      <Table name=\"customer\"/>\n"
```

Comment:
@mkambol 
it was failing for Oracle, 'cause in Oracle users and schemas are the same thing, and schema attribute is not filled then. 
Tested for Oracle and Mysql.